src/api-client.ts

```diff
@@ -167,11 +173,16 @@ export class RevAiApiClient {
         }
     }
 
-    async getTranscriptText(id: string): Promise<string> {
+    async getTranscriptText(id: string, fileName = null, fileLocation = null): Promise<string> {
```

Comment:
I think two parameters is a bit much to define the storage location for a file.
Can we combine the two into filePath? I think this will reduce confusion since the user doesn't have to worry about of the fileLocation must end in `/` or not.

Reply:
Have you thought about how the path must be specified? For example, can a user pass a relative path such as '~/user/username/myfile`?